src/AbstractPathResolver.js

```diff
@@ -15,6 +15,14 @@ export default class AbstractPathResolver {
   }
 
   async extendContext(...contexts) {
+    if (contexts.length) {
+      const defaultLanguages = contexts.map(context => context && context['@context'] && context['@context']['@language']).filter(Boolean);
+      if (defaultLanguages.length) {
+        this.defaultLanguage = defaultLanguages.pop();
+        if (defaultLanguages && defaultLanguages.some(language => language !== this.defaultLanguage))
```

Comment:
In what situation would `defaultLanguages` be undefined? I can't imagine any if you're popping it in the previous line